Tests/js/models/assets/ez-locationmodel-tests.js

```diff
@@ -256,6 +256,84 @@ YUI.add('ez-locationmodel-tests', function (Y) {
         }
     });
 
+    updateSortingTest = new Y.Test.Case({
+        name: "eZ location model update sorting tests",
+
+        setUp: function () {
+            this.capiMock = new Y.Mock();
+            this.capiGetService = 'getContentService';
+            this.locationId = '1/2/3';
+            this.sortField = "choucroute";
+            this.sortOrder = "couscous";
+            this.newSortField = 'SECTION';
+            this.newSortOrder = 'ASC';
+            this.contentServiceMock = new Y.Mock();
+            this.updateStruct = {
+                body: {
+                    LocationUpdate: {
+                        sortField: this.sortField,
+                        sortOrder: this.sortOrder
+                    }
+                }
+            };
+            this.model = new Y.eZ.Location({
+                id: this.locationId,
+                sortField: this.sortField,
+                sortOrder: this.sortOrder,
+            });
+            this.callbackCalled = false;
+
+            this.callback = Y.bind(function () {
+                this.callbackCalled = true;
+            }, this);
+
+            Mock.expect(this.capiMock, {
+                method: 'getContentService',
+                callCount: 2,
```

Comment:
it's better to avoid checking the number of call of a given methods. It's usually an implementation.